packages/template-retail-react-app/app/pages/account/wishlist/index.jsx

```diff
@@ -5,9 +5,14 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import React, {useState, useEffect, useRef} from 'react'
-import {Stack, Heading} from '@chakra-ui/layout'
```

Comment:
If we want to make sure that the chakra imports go through our `shared/ui` thing, then there's another candidate here: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/b2ed898f582093af4f4b7c9d49280ca97ac05827/packages/template-retail-react-app/app/components/_app/index.jsx#L31